bentoml/server/bento_api_server.py

```diff
@@ -107,15 +117,18 @@ def start(self):
         self.app.run(port=self.port, threaded=False)
 
     @staticmethod
-    def index_view_func():
+    def index_view_func(bento_service):
```

Comment:
consider refactor this to two different functions? `swagger_ui_func` and `user_defined_ui_func`, and decide which function to use in the `setup_routes` method?